src/windows.toit

```diff
@@ -77,6 +78,36 @@ to other paths.
 is-relative path/string -> bool:
   return not is-rooted path
 
+/**
+Convert the $path to an absolute path by prepending the current working directory to the path
+if it is not already absolute.
+
+The result is cleaned by $clean before being returned.
+*/
+to-absolute path/string -> string:
+  if is-absolute path: return clean path
+
+  if is-rooted path:
+    if is-relative-volume path:
+      split-path := split path
+      split-path[0] = "$split-path[0]$SEPARATOR"
+      return join split-path
+    else:
+      if is-absolute-volume directory.cwd:
```

Comment:
Would a `\foo` not work if the cwd is a UNC path?
Is the `if` necessary?

Reply:
I think in that case it will use the final return in the method. I added a comment for clarification